package.json

```diff
@@ -17,82 +17,45 @@
     "modules/*"
   ],
   "scripts": {
-    "bootstrap": "scripts/bootstrap.sh",
-    "build": "lerna run build",
-    "clean": "lerna clean --yes && rm -rf node_modules",
-    "clean-all": "npm run clean && find . -name yarn.lock -exec rm -fr {} \\; && find . -name node_modules -exec rm -fr {} \\;",
-    "cover": "NODE_ENV=test nyc --reporter html --reporter cobertura --reporter=lcov npm run test cover",
-    "lint": "scripts/lint.sh",
-    "lint-docs": "scripts/lint-markdown.sh",
-    "publish-prod": "npm run bootstrap && npm run build && npm run test && npm run test dist && scripts/publish.sh prod",
-    "publish-beta": "npm run bootstrap && npm run build && npm run test && npm run test dist && scripts/publish.sh beta",
+    "bootstrap": "yarn && ocular-bootstrap",
+    "clean": "ocular-clean",
+    "build": "ocular-clean && ocular-build",
+    "version": "ocular-build core,main",
+    "lint": "ocular-lint",
+    "cover": "ocular-test cover",
+    "publish-prod": "ocular-publish prod",
```

Comment:
Does ocular-publish work now? it doesn't work when we published math.gl last time.

Reply:
A lot of changes have been made to the ocular-dev-tools module. Please report any bugs in the ocular repo.